docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -872,6 +876,38 @@ printf "{\n%s\n}" "${key_values::-1}"
 
 The only parameter of the script is a file containing the flags, with each flag on its own line.
 
+### Backfill
+
+Backfill command uploads Prometheus TSDB blocks into Grafana Mimir, by using block upload API exposed by compactor component.
+
+If the command is interrupted it can be started again. Mimirtool will detect which blocks are already uploaded, and will only upload unfinished or new blocks.
+
+If block upload is not enabled for the user, `mimirtool backfill` will fail.
```

Comment:
I would rephrase it, saying as first thing that block upload is disabled by default. Then how to enable it (link to config doc page), and finally what happens if you try to use it with `mimirtool` but it's disabled (even if I guess at that point it will be obvious).

Reply:
Thanks, done.